modules/@apostrophecms/image/ui/apos/components/AposMediaManagerEditor.vue

```diff
@@ -124,6 +125,12 @@ export default {
         return [];
       }
     },
+    isModified: {
+      type: Boolean,
+      default() {
+        return false;
+      }
```

Comment:
```suggestion
      default: false
```

since it's a primitive type we're setting, no need for a function (minor stuff 🥖 )